src/pages/PrivateNotes/PrivateNotesEditPage.js

```diff
@@ -114,6 +114,7 @@ function PrivateNotesEditPage({route, personalDetailsList, session, report}) {
                 <HeaderWithBackButton
                     title={translate('privateNotes.title')}
                     subtitle={translate('privateNotes.myNote')}
+                    onBackButtonPress={() => Navigation.goBack(ROUTES.PRIVATE_NOTES_VIEW.getRoute(report.repotID, route.params.accountID))}
```

Comment:
`repotID` lol 😅 
we're fixing it [here](https://github.com/Expensify/App/issues/30211)